File: 1 - INICIANTE/python/1048.py
salario = float(input())
novo_salario, valor_reajuste, percentual= float(), float, int()


if 0<=salario<=400.00:
    novo_salario = salario*1.15
    valor_reajuste = salario*1.15-salario
    percentual = 15

elif 400.00<salario<=800.00:
    novo_salario = salario * 1.12
    valor_reajuste = salario * 1.12 - salario
    percentual = 12

elif 800.00<salario<=1200.00:
    novo_salario = salario * 1.10
    valor_reajuste = salario * 1.10 - salario
    percentual = 10

elif 1200.00<salario<=2000.00:
    novo_salario = salario * 1.07
    valor_reajuste = salario * 1.07 - salario
    percentual = 7

else:
    novo_salario = salario * 1.04
    valor_reajuste = salario * 1.04 - salario
    percentual = 4

print('''Novo salario: {:.2f}
Reajuste ganho: {:.2f}
Em percentual: {} %'''.format(novo_salario, valor_reajuste, percentual))

File: 1 - INICIANTE/C++17/1048.cpp
#include <iostream>
#include <iomanip>
#include <cmath>
using namespace std;

int main() {
 
 float x,salario,reajuste;
 int y;
   // y = percentual do reajuste
cin >> x;


if (x<=400) {
  salario = (x*0.15)+x;
  reajuste = x*0.15;
  y = 15;
}

else if (x>=400.01 && x<=800.00) {
  salario = (x*0.12)+x;
  reajuste = x*0.12;
  y = 12;
}

else if (x>=800.01 && x<=1200.00) {
  salario = (x*0.10)+x;
  reajuste = x*0.10;
  y = 10;
}

else if (x>=1200.01 && x<=2000.00) {
  salario = (x*0.07)+x;
  reajuste = x*0.07;
  y = 7;
}

else {
  salario = (x*0.04)+x;
  reajuste = x*0.04;
  y = 4;
}

cout << fixed << setprecision(2) << "Novo salario: " << salario << "\n" << "Reajuste ganho: " << reajuste << "\n" << "Em percentual: " << y << " %" << "\n";

}